bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -177,11 +181,13 @@ public void reset()
   public void purge(final int baseSeconds, final int windowId)
   {
     final long longWindowId = (long)baseSeconds << 32 | windowId;
-    logger.debug("Purging {} from window ID {} to window ID {}", this, Codec.getStringWindowId(first.starting_window), Codec.getStringWindowId(longWindowId));
+    logger.debug("Purging {} from window ID {} to window ID {}", this, Codec.getStringWindowId(first.starting_window),
+        Codec.getStringWindowId(longWindowId));
 
     int numberOfInMemBlockPurged = 0;
     synchronized (this) {
-      for (Block prev = null, temp = first; temp != null && temp.starting_window <= longWindowId; prev = temp, temp = temp.next) {
+      for (Block prev = null, temp = first; temp != null && temp.starting_window <= longWindowId;
```

Comment:
It depends on the width of the monitor. When I don't use external monitor, wrapping at 120 char length improves readability.